ops/ccs-ops-misc/load_test/high_volume_suite.py

```diff
@@ -0,0 +1,374 @@
+"""High Volume Load test suite for BFD Server endpoints."""
+
+from typing import Dict, List
+
+from locust import events, tag, task
+from locust.env import Environment
+
+from common import data, db, validation
+from common.bfd_user_base import BFDUserBase
+from common.locust_utils import is_distributed, is_locust_master
+from common.url_path import create_url_path
+
+MASTER_BENE_IDS: List[str] = []
+MASTER_CONTRACT_DATA: List[Dict[str, str]] = []
+MASTER_HASHED_MBIS: List[str] = []
+
+
+@events.test_start.add_listener
+def _(environment: Environment, **kwargs):
+    if (
+        is_distributed(environment)
+        and is_locust_master(environment)
+        or not environment.parsed_options
+    ):
+        return
+
+    # See https://docs.locust.io/en/stable/extending-locust.html#test-data-management
+    # for Locust's documentation on the test data management pattern used here
+    global MASTER_BENE_IDS
+    MASTER_BENE_IDS = data.load_from_parsed_opts(
+        environment.parsed_options,
+        db.get_bene_ids,
+        use_table_sample=True,
+        data_type_name="bene_ids",
+    )
```

Comment:
We should keep in mind that adjustments to the data load method will affect both suites.  (I assume @malessi will be creating a new method to pull specific data especially for the regression suite, so we dont inadvertently affect this one in the future?)

Reply:
We might need to make some adjustments to data loading that are specific to the high volume suite as well. We can decouple them when the time comes/as needed.